R/osmactive.R

```diff
@@ -10,6 +10,12 @@ et_active = function() {
     "cycleway:left",
     "cycleway:right",
     "cycleway:both",
+    "cycleway:left:bicycle",
+    "cycleway:right:bicycle",
+    "cycleway:both:bicycle",
+    "cycleway:left:segregated",
+    "cycleway:right:segregated",
+    "cycleway:both:segregated",
```

Comment:
Good catch.

I assume the test example was:
https://www.openstreetmap.org/way/1101640529

Be aware that the data there is actually incorrect. The `segregation` tag refers to segregation from pedestrians, not traffic (this was always a badly-named tag). That doesn't make any sense in the context of `cycleway=lane` because that is by definition a cycle lane on a road and cannot be shared with pedestrians. So really that `cycleway:left:segregated` is meaningless, and those two `segregation` tags should be deleted.  The `separation` subkey usage is, however, correct, and is quite new.

https://wiki.openstreetmap.org/wiki/Bicycle#Separation_from_other_road_users

In your code, I would apply `segregation` and its variants only to things marked with `cycleway=track` as that is the only context in which it should be used.

Reply:
Correct, how did you know?

https://www.openstreetmap.org/way/1101640529

Reply:
> noticed that Headingley Lane

This gave me a clue :)